Source/Scene/GroundPrimitive.js

```diff
@@ -74,6 +74,12 @@ define([
      * is supported at this time.
      * </p>
      * <p>
+     * Because of the cutting edge nature of this feature in WebGL, it requires the EXT_frag_depth extension, which is currently only supported in Chrome,
+     * Firefox, and Edge. Apple support is expected in iOS 9 and MacOS Safari 9. Android support varies by hardware and IE11 will most likely never support
```

Comment:
That sucks.  I just confirmed with @klim705 that frag depth is supported on desktop Safari.  I'll check my iPad when I get home tonight.